pkg/ccl/backupccl/schedule_exec.go

```diff
@@ -439,7 +439,9 @@ func unlinkDependentSchedule(
 
 	// Clear the DependentID field since we are dropping the record associated
 	// with it.
+	// TODO(benbardin): This is probably surprising behavior, rethink.
```

Comment:
Should we reword this TODO with a link to https://github.com/cockroachdb/cockroach/issues/87435 instead? As is this TODO isn't very informative so we should flesh it out depending on what you decide.

Reply:
Yup, fixed.